### PR TITLE
DDWR-1132 Fix updated timestamp to reflect created/updated timestamp.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: warehouse/sql/V1_1_0_22__fix_accommodation_update.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+USE ${schemaName};
+
+UPDATE accommodation 
+ SET updated = now()
+WHERE updated is null;
```
Comment (agorina) [inline warehouse/sql/V1_1_0_22__fix_accommodation_update.sql]: It did not update the timestamp on create.
Another alternative is to find the latest import with content CODE and use a timestamp from there. 
Any preferences?

Comment (wtritch) [reply warehouse/sql/V1_1_0_22__fix_accommodation_update.sql]: I think setting it to "now" is completely reasonable.

File: warehouse/sql/V1_1_0_22__fix_accommodation_update.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+ALTER TABLE accommodation MODIFY COLUMN updated TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6);
```
Comment (malaffoon) [inline warehouse/sql/V1_1_0_22__fix_accommodation_update.sql]: foolish consistency should've caught this ... oops

Comment (agorina) [reply warehouse/sql/V1_1_0_22__fix_accommodation_update.sql]: true :)

**Pull request merged**